packages/nameguard-react/src/components/Report/LoadingSkeleton.tsx

```diff
@@ -22,7 +22,7 @@ export const LoadingSkeleton = ({ parsedName }: LoadingSkeletonProps) => {
           </p>
           {name ? (
             <>
-              <h1 className="mt-1 text-2xl md:text-4xl text-black font-semibold md:font-bold overflow-hidden overflow-ellipsis font-ens-name">
+              <h1 className="mt-1 text-2xl md:text-4xl text-black font-semibold md:font-bold overflow-hidden overflow-ellipsis ens-webfont">
```

Comment:
This makes me sad haha. Hopefully we can bridge the custom font pkg and tailwind with a custom tailwind plugin so we can keep to their conventions

Reply:
What I mean is that we're hooking into the provided classes instead of classes built by Tailwind. There's no concern here but something we can improve later if we create a tailwind plugin for the font family.

Reply:
Hey guys! I've created [this story](https://app.shortcut.com/ps-web3/story/23674/update-namehash-ens-webfont-architecture) to proceed with this enhancement soon!